.github/workflows/ci.yml

```diff
@@ -0,0 +1,58 @@
+name: Numtracker CI
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: dtolnay/rust-toolchain@stable
+      with:
+          toolchain: nightly
+          components: rustfmt, clippy
+    - uses: Swatinem/rust-cache@v2
+    - name: Check formatting
+      run: cargo +nightly fmt -- --check
+    - name: Clippy
+      run: |
+        cargo --version
+        cargo clippy --version
+        cargo clippy --all-targets --all-features -- --deny warnings
+
+  sqlx_offline:
```

Comment:
Could: Give this check a more descriptive name, maybe `sql queries` (if I'm reading what it does right)?

Reply:
It compares the query data used for building the project offline with the current schema - so kind of both? [Docs here](https://github.com/launchbadge/sqlx/tree/main/sqlx-cli#enable-building-in-offline-mode-with-query)